File: Leetcode/c/2125.c
int numberOfBeams(char** bank, int bankSize) {
    int output = 0, prev = 0;;
    for (int i = 0; i < bankSize; ++i) {
        int count = 0;
        for (int j = 0; j < strlen(bank[i]); ++j) {
            if (bank[i][j] == '1') {
                count++;
            }
        }
        if (count > 0) {
            output += (prev * count);
            prev = count;
        }
    }
    return output;
}

File: Leetcode/Python/2125.py
class Solution:
    def numberOfBeams(self, bank: List[str]) -> int:
        output = 0
        prev = 0
        for row in bank:
            count = 0
            for c in row:
                if c == "1":
                    count += 1
            if count:
                output += count * prev
                prev = count
        return output
